### PR TITLE
Add recipe for pe-utils/fw-tools/testnet.sh (v 2.0.11)

### PR DESCRIPTION
**Pull request by JanneKiiskila**
For means to troubleshoot/verify the connections do not get blocked by a firewall.

https://github.com/PelionIoT/pe-utils/tree/master/fw-tools

Ref: IZUMABL-200

### CODE REVIEW

File: recipes-misc/testnet/testnet_2.0.0.bb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+PR = "r0"
+
+DEPENDS = ""
+RDEPENDS_${PN} += "busybox openssl"
+
+RM_WORK_EXCLUDE += "${PN}"
+
+FILES_${PN} = "\
+${EDGE_BIN}/ \
+/etc/tmpfiles.d/userdata-tmpfiles.conf \
```
Comment (JanneKiiskila) [inline recipes-misc/testnet/testnet_2.0.0.bb:24]: This doesn't look quite right?

Comment (petedyerarm) [reply recipes-misc/testnet/testnet_2.0.0.bb:24]:
```suggestio
```

Comment (petedyerarm) [inline recipes-misc/testnet/testnet_2.0.0.bb:24]:
```suggestion

```

**Review by petedyerarm — CHANGES_REQUESTED** [review]

File: recipes-misc/testnet/testnet_2.0.0.bb (hunk 1 above)
Comment (petedyerarm) [reply recipes-misc/testnet/testnet_2.0.0.bb:24]: testnet needs to be added to https://github.com/PelionIoT/meta-pelion-edge/blob/testnet/recipes-core/images/console-image.bb to be included in the image.

Comment (JanneKiiskila) [reply recipes-misc/testnet/testnet_2.0.0.bb]: Yep, I have done those already -I just forgot them from this PR last night. They are now in.
We have actually TWO of those files, why two?

Comment (JanneKiiskila) [reply recipes-misc/testnet/testnet_2.0.0.bb]: `meta-lmp-support/recipes-core/images/console-image-lmp.bb` and `recipes-core/images/console-image.bb` - which one do we really use? Or do we somehow use both?

Comment (petedyerarm) [reply recipes-misc/testnet/testnet_2.0.0.bb]: I suspect that the one in recipes-core is for a non-LmP Yocto build, but I doubt we support that anymore.

Comment (JanneKiiskila) [reply recipes-misc/testnet/testnet_2.0.0.bb]: Okay, that would definitely explain it.

**Comment by JanneKiiskila** [comment]
```
fio@raspberrypi3-64:~$ testnet.sh
[   OK   ]      ping bootstrap.us-east-1.mbedcloud.com
[   OK   ]      ping lwm2m.us-east-1.mbedcloud.com
[   OK   ]      netcat bootstrap.us-east-1.mbedcloud.com 443
[   OK   ]      netcat bootstrap.us-east-1.mbedcloud.com 5684
[   OK   ]      netcat lwm2m.us-east-1.mbedcloud.com 443
[   OK   ]      netcat lwm2m.us-east-1.mbedcloud.com 5684
[   OK   ]      netcat k8s.us-east-1.mbedcloud.com 443
[   OK   ]      netcat gateways.us-east-1.mbedcloud.com 443
[   OK   ]      TLS to bootstrap server (port 5684)
[   OK   ]      TLS to LwM2M server (port 5684)
[   OK   ]      TLS to bootstrap server (port 443)
[   OK   ]      TLS to LwM2M server (port 443)
[   OK   ]      TLS to k8s server (port 443)
[   OK   ]      TLS to gateway server (port 443)
```

**Pull request merged**